101-backup-vault-basic/azuredeploy.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vaultName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Vault"
+      }
+    },
+    "vaultStorageRedundancy": {
+      "type": "string",
+      "defaultValue": "GeoRedundant",
+      "allowedValues": [
+        "LocallyRedundant",
+        "ZonallyRedundant",
+        "GeoRedundant"
+      ],
+      "metadata": {
+        "description": "Change Vault Storage Type (not allowed if the vault has registered backups)"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.DataProtection/BackupVaults",
+      "apiVersion": "2020-01-01-alpha",
+      "name": "[parameters('vaultName')]",
+      "location": "[parameters('location')]",
+      "identity": {
+        "type": "systemAssigned"
```

Comment:
how is this used?

Reply:
to configure backup to any workload, Vault's managed identity needs to be granted necessary permissions on the workload. For that identity must be enabled at first place.